source/ConversationPanel.cpp

```diff
@@ -470,3 +504,13 @@ Point ConversationPanel::Paragraph::Draw(Point point, const Color &color) const
 	point.Y() += wrap.Height();
 	return point;
 }
+
+
+
+int ConversationPanel::MapChoice(int n) const
```

Comment:
Style fix: Functions in the .cpp file should match their order in the header file. So this should go below ConversationPanel::ClickChoice (assuming that's also where you move it in the header file). The comment I mentioned that should be added in the header should also appear here.